scripts/gha/test_simulator.py

```diff
@@ -129,7 +137,7 @@
     "android_sdk", "system-images;android-29;google_apis;x86",
     "See module docstring for details on how to set and get this id.")
 flags.DEFINE_string(
-    "build_tools_version", "28.0.3",
+    "build_tools_version", "29.0.2",
```

Comment:
Isn't this value in `build.gradle`? Is this the right place to update the config value like this?

Reply:
We are using tools under `$ANDROID_HOME/build-tools/${build_tools_version}`